frontend/src/app/sessions/session/session.component.html

```diff
@@ -0,0 +1,109 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+<div *ngIf="!selectedSessions.length">
+  <span
+    >Please select the sessions that you'd like to open in the session
+    viewer:</span
+  >
+  <div *ngIf="cachedSessions && !cachedSessions.length">
+    No sessions found. Please create a session in the 'Sessions' tab first.
+  </div>
+  <div *ngIf="cachedSessions === undefined">
+    <ngx-skeleton-loader
+      *ngFor="let _ of [0, 1, 2, 3]"
+      appearance="circle"
+      [theme]="{
+        'border-radius': '5px',
+        height: '24px',
+        width: '80%',
+        border: '1px solid white',
+        margin: '0'
+      }"
+    ></ngx-skeleton-loader>
+  </div>
+  <div class="flex flex-col" *ngIf="cachedSessions && cachedSessions.length">
+    <mat-checkbox
+      class="w-fit"
+      [disabled]="!sessionService.beautifyState(session.state).success"
+      matTooltip="Please wait until the session is running. You can check the status in the 'Sessions' tab."
+      [matTooltipDisabled]="sessionService.beautifyState(session.state).success"
+      [value]="session.id"
+      (change)="changeSessionSelection($event, session)"
+      *ngFor="let session of cachedSessions"
+      >{{ session.version?.tool?.name }} {{ session.version?.name }},
+      {{ session.type
+      }}<span *ngIf="session.type === 'readonly'">
+        (project {{ session.project!.name }})</span
+      >, created at {{ session.created_at | date: "EE, dd MMM y HH:mm:ss" }}
+    </mat-checkbox>
+  </div>
+  <div class="border-2 p-2 mt-2 rounded w-fit">
+    Please note: We're not technically preventing that you select two Guacamole
+    sessions (for example two Capella sessions), <br />but the Guacamole server
+    will not let you connect to the sessions. We're working on making this
+    possible.
```

Comment:
I would phrase this differently. Two sessions using Eclipse based tooling.

It's completely transparent to the users that we use Guacamole to expose the sessions.